src/core_plugins/getting_started/public/components/getting_started/getting_started.js

```diff
@@ -0,0 +1,63 @@
+import { uiModules } from 'ui/modules';
+import uiChrome from 'ui/chrome';
+import 'ui/storage';
+import 'ui/filters/trust_as_html';
+import { gettingStartedRegistry } from 'ui/getting_started_registry';
+import { GETTING_STARTED_OPT_OUT } from '../../lib/constants';
+
+import kibanaLogo from 'ui/images/logo-kibana-small.svg';
+import beatsLogo from 'ui/images/logo-beats-small.svg';
+import logstashLogo from 'ui/images/logo-logstash-small.svg';
+import dashboardIcon from 'ui/images/icon-dashboard.svg';
+import shieldIcon from 'ui/images/icon-shield.svg';
+
+import template from './getting_started.html';
+import './getting_started.less';
+
+const app = uiModules.get('kibana');
+
+app.directive('gettingStarted', function ($injector) {
+
+  const localStorageService = $injector.get('localStorage');
+
+  return {
+    restrict: 'E',
+    template: template,
+    scope: {
+    },
+    bindToController: true,
+    controllerAs: 'gettingStarted',
+    controller: class GettingStartedController {
+      constructor() {
+        if (this.hasOptedOut) {
```

Comment:
@spalger Perhaps you have some idea of how to pull off what we are trying here?

We want to not show the left nav when the Getting Started page (introduced in this PR) is shown to the user. Right now, there's a flash of the left nav before it gets hidden. Any way we could avoid that?